docs/developer_guide/lane_pose_calculation/LanePoseCalculation.md

```diff
@@ -0,0 +1,83 @@
+
+!!! Note
+    It is sufficient to check [this document](UpdateFrame.md) and [this section](#lane-coordinate-system-calculation-algorithm-for-a-specific-lane), if you are not a simulator developer but a scenario writer. That is because the distance measurement via condition of the OpenSCENARIO is done after frame update basically.
+
+## Lane coordinate system calculation algorithm for a specific lane
+
+!!! summary
+    - The calculation of the pose in the lane coordinate system involves two steps: first, filtering by "which lanes can be matched," and then "calculating the pose in the specific lane coordinate system.  
+    - The parameter that most affects the matching results is the length of the horizontal bar. This length depends on the type of entity and the timing of the calculation. If you want to check the length quickly, please check [this section](#quick-guide-to-horizontal-bar-lengths).
+
+### Quick guide to horizontal bar lengths 
+
+The unit of the table is meter.
+
+| EntityType \ Timing of calculation | [Spawn](Spawn.md) | [UpdateFrame](UpdateFrame.md)                                                          | [Getting longitudinal distance](GetLongitudinalDistance.md) |
+| ---------------------------------- | ----------------- | -------------------------------------------------------------------------------------- | ----------------------------------------------------------- |
+| EgoEntity                          | 2.0               | [(tread of the entity) + 2.0](UpdateFrame.md#calculate-pose-in-lane-coordinate-system) | 10                                                          |
+| VehicleEntity                      | 2.0               | N/A                                                                                    | 10                                                          |
+| PedestrianEntity                   | 2.0               | [2.0 or 4.0](UpdateFrame.md#pedestrian-entity-with-behavior-tree)                      | 10                                                          |
+| MiscObject                         | 2.0               | N/A                                                                                    | 10                                                          |
+
+### Detail of the lane coordinate system calculation algorithm for a specific lane
+
+After the lanes to be matched have been determined, the calculation of the specific lane coordinate system to be performed is always based on a per spline curve and line segment determination.
+
+![Lane pose calculation](../../image/lane_pose_calculation.png "Lane pose calculation.")
+
+The centerline of a lane is expressed by the cubic equation [Catmull-Rom spline.](https://en.wikipedia.org/wiki/Centripetal_Catmull%E2%80%93Rom_spline)  
+Catmull-Rom spline and line perturbation are implemented by finding the multiple solution of one cubic equation.  
```

Comment:
Does `line perturbation` have the same meaning as `line segment` elesewhere?
If they mean the same thing, please use the same expression.
(Also, the expressions used in the figure and text are different, making it difficult to understand...)

Reply:
I fixed to `line segment` in https://github.com/tier4/scenario_simulator_v2/pull/1192/commits/929775a6c2974f8ce9229bd9b5a215249beced5b